classes/options-general.php

```diff
@@ -328,51 +328,51 @@ function print_page() {
 
 													<tr valign="top">
 														<td width="150"><label
-																for="address_format"><?php _e( 'Address Format', 'simplemap' ); ?></label>
+																for="address_format"><?php _e( 'Address Format', 'SimpleMap' ); ?></label>
 														</td>
 														<td>
 															<select id="address_format" name="address_format">
 																<option
-																	value="town, province postalcode" <?php selected( $address_format, 'town, province postalcode' ); ?>><?php echo '[' . __( 'City/Town', 'simplemap' ) . '], [' . __( 'State/Province', 'simplemap' ) . ']&nbsp;&nbsp;[' . __( 'Zip/Postal Code', 'simplemap' ) . ']'; ?></option>
+																	value="town, province postalcode" <?php selected( $address_format, 'town, province postalcode' ); ?>><?php echo '[' . __( 'City/Town', 'SimpleMap' ) . '], [' . __( 'State/Province', 'SimpleMap' ) . ']&nbsp;&nbsp;[' . __( 'Zip/Postal Code', 'SimpleMap' ) . ']'; ?></option>
 
 																<option
-																	value="town province postalcode" <?php selected( $address_format, 'town province postalcode' ); ?>><?php echo '[' . __( 'City/Town', 'simplemap' ) . ']&nbsp;&nbsp;[' . __( 'State/Province', 'simplemap' ) . ']&nbsp;&nbsp;[' . __( 'Zip/Postal Code', 'simplemap' ) . ']'; ?></option>
+																	value="town province postalcode" <?php selected( $address_format, 'town province postalcode' ); ?>><?php echo '[' . __( 'City/Town', 'SimpleMap' ) . ']&nbsp;&nbsp;[' . __( 'State/Province', 'SimpleMap' ) . ']&nbsp;&nbsp;[' . __( 'Zip/Postal Code', 'SimpleMap' ) . ']'; ?></option>
 
 																<option
-																	value="town-province postalcode" <?php selected( $address_format, 'town-province postalcode' ); ?>><?php echo '[' . __( 'City/Town', 'simplemap' ) . '] - [' . __( 'State/Province', 'simplemap' ) . ']&nbsp;&nbsp;[' . __( 'Zip/Postal Code', 'simplemap' ) . ']'; ?></option>
+																	value="town-province postalcode" <?php selected( $address_format, 'town-province postalcode' ); ?>><?php echo '[' . __( 'City/Town', 'SimpleMap' ) . '] - [' . __( 'State/Province', 'SimpleMap' ) . ']&nbsp;&nbsp;[' . __( 'Zip/Postal Code', 'SimpleMap' ) . ']'; ?></option>
 
 																<option
-																	value="postalcode town-province" <?php selected( $address_format, 'postalcode town-province' ); ?>><?php echo '[' . __( 'Zip/Postal Code', 'simplemap' ) . ']&nbsp;&nbsp;[' . __( 'City/Town', 'simplemap' ) . '] - [' . __( 'State/Province', 'simplemap' ) . ']'; ?></option>
+																	value="postalcode town-province" <?php selected( $address_format, 'postalcode town-province' ); ?>><?php echo '[' . __( 'Zip/Postal Code', 'SimpleMap' ) . ']&nbsp;&nbsp;[' . __( 'City/Town', 'SimpleMap' ) . '] - [' . __( 'State/Province', 'SimpleMap' ) . ']'; ?></option>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 339 characters

<!-- Comment by [SideCI](https://sideci.com) -->